### ISSUE TITLE
Angular 2

### ISSUE THREAD

**Issue by Florian79** [a5hik/ng-sortable#291]
are there plans to migrate ng-sortable to Angular 2 or do you have a migration guide/recommendations?

**Comment by a5hik** [comment]
@Florian79 yes definitely we look forward to upgrade to angular2 support. But at the moment there is no support this.

**Comment by a5hik** [comment]
Coming up with a new version of this library completely written in Angular2 from scratch...

https://github.com/a5hik/angular-sortable

**Comment by lluiscanals** [comment]
Any updates on the Angular 2 version? Do you any estimations on when it will be available?

Thanks!

**Comment by a5hik** [comment]
I have the version already available.. let me push to the repo.. in a few
days.

- Ashik

**Comment by ibaivalencia** [comment]
Hi @a5hik , anything new about this release for angular 2?
Thanks! :)

**Comment by lluiscanals** [comment]
Yes please :)

**Comment by thardy-pd** [comment]
I would be willing to help out with the Angular 2/4 version.

**Comment by a5hik** [comment]
Hi I already have a working version of it. .. will publish it soon.
I was just thinking whether to host in a private mode or not.

- Ashik

**Comment by lluiscanals** [comment]
Hi,
do you have any news on this?

Thanks and regards.

**Comment by johankvint** [comment]
Hi,

Just wanted to check if there was any progress on this.
I'm updating my project to angular 5 and used this for drag'n'drop for angular 1.
Any chance this will be released soon so I can wait, otherwise I will have to choose another solution.

BR

**Comment by a5hik** [comment]
Hi I'm looking ways to publish the lib and adding a demo app for same. watch the repo for updates..
https://github.com/a5hik/angular-sortable

**Comment by thardy** [comment]
How about just npm, like ng-sortable?  Not sure I understand.  I see you have angular-sortable in the npmjs registry.  That sounds like a great name.  Most of the npm modules for angular 2+ are prefixed by "angular-", so that would work nicely.

If you're thinking about a private module and charging for it, I can't be of help there.  I haven't had much luck with any of those.  Most don't make it very far, but I've seen a few that had open source modules, then they had more feature-rich versions available via some marketplace.

**Comment by a5hik** [comment]
Hi it's open source, and I don't have any plans to make it private. the lib is already in a working state with the demo app.. https://a5hik.github.io/angular-sortable/
I just need to publish it as npm, otherwise for time being you can checkout the repo and you see the source of the lib with a demo app.
